samples/throttling/Version_6/Sample/Program.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+public class Program
+{
+    static void Main(string[] args)
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        BusConfiguration configuration = new BusConfiguration();
+        configuration.EndpointName("Samples.Throttling");
+        configuration.UsePersistence<InMemoryPersistence>();
+        configuration.SendFailedMessagesTo("error");
+        configuration.LimitMessageProcessingConcurrencyTo(1);
```

Comment:
@timbussmann removing this causes the console output to go to hell... for the sample we could just limit to 1